cinema/models.py

```diff
@@ -1,6 +1,11 @@
+import pathlib
+import uuid
+from tkinter import image_names
```

Comment:
The import `from tkinter import image_names` seems unnecessary and unrelated to the rest of the code. Consider removing it if it's not used elsewhere in your project.